### PR TITLE
TUNIC: Update game page for blurb about playing vanilla first

### PR DESCRIPTION
**Pull request by ScipioWright**
## What is this fixing or adding?
Add a section to the game page discouraging the player from playing the rando before playing vanilla, as that is a terrible idea.
Also changing settings -> options

## How was this tested?
Reading

## If this makes graphical changes, please attach screenshots.
N/A

### CODE REVIEW

**Review by hatkirby — APPROVED** [review]

**Review by ReverM — CHANGES_REQUESTED** [review]
There is spoiler material earlier in the page. I would move it above. Before the point that tells what is randomized ideally

**Review by silent-destroyer — APPROVED** [review]

**Review by ReverM — APPROVED** [review]
LGTM

File: worlds/tunic/docs/en_Tunic.md [hunk 1]
```diff
@@ -1,15 +1,20 @@
...
 
-## Where is the settings page?
+## Where is the options page?
 
-The [player settings page for this game](../player-settings) contains all the options you need to configure and export a config file.
+The [player options page for this game](../player-options) contains all the options you need to configure and export a config file.
+
+## I haven't played TUNIC before.
+
+**Play vanilla first.** It is **_heavily discouraged_** to play this randomizer before playing the vanilla game, moreso than most other games in Archipelago.
```
Comment (black-sliver) [inline worlds/tunic/docs/en_Tunic.md:9]:
```suggestion
**Play vanilla first.** It is **_heavily discouraged_** to play this randomizer before playing the vanilla game.
```
I think this is bad. You probably haven't played them all and you can't possibly know which games will come next.

**Review by black-sliver — APPROVED** [review]
Thanks!

**Pull request merged**